tdrs-backend/docs/api/report.md

```diff
@@ -0,0 +1,64 @@
+# Report metadata
+
+Accepts s3 slugs and maintains every prior version of the file
```

Comment:
can you say more about how this endpoint is used in the application and how the s3 slugs are used? @riatzukiza

Reply:
@ADPennington I have updated this